package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Integration between React Router and React-Bootstrap",
   "main": "./lib/index.js",
   "scripts": {
+    "update-dep": "./node_modules/.bin/npm-check-updates -u",
```

Comment:
the `./node_modules/.bin/` is unecessary anything in that directory is automatically in the PATH when running an npm script so `.npm-check-updates -u` is enough